resources/test/valid/chainspec.toml

```diff
@@ -219,6 +220,7 @@ max_ttl = '35years'
 max_dependencies = 36
 max_block_size = 37
 block_max_deploy_count = 375
+block_max_transfer_count = 375
```

Comment:
Is this file just for a deserialization test? Then it's probably better to never repeat any values, and make this `376`.